__tests__/app.test.js

```diff
@@ -264,6 +264,7 @@ describe("/api/articles/:article_id", () => {
               "created_at",
               "votes",
               "article_img_url",
+              "comment_count",
```

Comment:
We should create a new test and not modify the old one, which should be robust enough to not fail upon modification of the query :)